syntax = "proto3";

package example.simple;

option go_package = "basics5-protobuf-go/proto";

message SimpleMessage {
  uint32 id = 1;
  bool is_simple = 2;
  string name = 3;
  repeated sint32 numbers = 4;
}
